app/pages/settings/email.jsx

```diff
@@ -292,12 +313,27 @@ class EmailSettingsPage extends React.Component {
                   <i className='fa fa-times-circle' style={{ color: '#e35950' }} />
                   {' '}
                   <Translate content="emailSettings.general.emailUnverified" />
-                  {/*
                   {' | '}
-                  <a href="#" onClick={this.requestConfirmationEmail}>
-                    <Translate content="emailSettings.general.emailUnverifiedPrompt" />
-                  </a>
-                  */}
+                  {(this.state.requestConfirmationEmailStatus === 'idle') && (
+                    <a href="#" onClick={this.requestConfirmationEmail}>
+                      <Translate content="emailSettings.general.emailUnverifiedPrompt" />
+                    </a>
+                  )}
+                  {(this.state.requestConfirmationEmailStatus === 'posting') && (
+                    <span>
+                      <Translate content="emailSettings.general.emailUnverifiedRequesting" />
+                    </span>
+                  )}
+                  {(this.state.requestConfirmationEmailStatus === 'success') && (
+                    <span>
+                      <Translate content="emailSettings.general.emailUnverifiedSuccess" />
+                    </span>
+                  )}
+                  {(this.state.requestConfirmationEmailStatus === 'error') && (
+                    <span style={{ color: 'red' }}>
+                      <Translate content="emailSettings.general.emailUnverifiedError" />
+                    </span>
+                  )}
                 </div>
```

Comment:
Since the text here is changing, you’ll want to wrap it in one of the live region roles (probably ’status’), so that changes are announced in screen readers. 

Note that live regions are expected to be empty when they load/mount, so any initial status won’t be read out.

Reply:
I forgot to mention: the ‘Verify my email’ button should **not** be part of the live region. That would be weird. Something like this should work without discriminating against disabled people:

```jsx
<button onClick={verifyEmail}>
  Verify your email address
</button>
<span role=‘status’>
/*
  Changes to verification status
  will be announced here. The initial
  status must be an empty string. 
*/
  {verificationRequestStatus}
</span>

Reply:
The HTML `<output>` element is also supported in all browsers, for live text. 

https://www.scottohara.me/blog/2019/07/10/the-output-element.html

Reply:
👍 All status messages have now been placed in an `<output>`